webplugin/js/app/kommunicate.js

```diff
@@ -116,6 +116,42 @@ $applozic.extend(true, Kommunicate, {
             }, SET_TIMEOUT_DURATION);
         }
     },
+    groupSpecificData: function (userSpecificMetadata) {
+        let [type, metaType, groupId,metadataToShow] = [
```

Comment:
use var in chat widget code, let might not work in IE
in dashboard code, it is fine to use let